models/config.go

```diff
@@ -3,11 +3,11 @@ package models
 var PathConfig = "/configurations"
 
 type ConfigBodyPost struct {
-	OidcVerifyCert        bool   `json:"oidc_verify_cert,omitempty"`
```

Comment:
The omitempty is required. As this struct is used for other things

Reply:
I did this, because with the omitempty, no boolean value are working

Reply:
I think that removing the `omitempty` from all boolean values are fine. As it should only affect the boolean fields if we were using a pointer to `bool`